features/promotion/managing_catalog_promotions/creating_catalog_promotion.feature

```diff
@@ -158,3 +158,20 @@ Feature: Creating a catalog promotion
         And I add a new catalog promotion action
         Then I should see the catalog promotion scope configuration form
         And I should see the catalog promotion action configuration form
+
+    @api @ui @javascript
+    Scenario: Adding a new catalog promotion that applies on variant with exclusive promotion
```

Comment:
The catalog promotion you are creating here is not exclusive (if I see correctly), but you should create an exclusive one. I would be for 2 scenarios, one for creating an exclusive promotion that has bigger priority than the existing promotion, and one with a lower priority